# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditorBase.cpp
# Commit: 0bffa7979736
# Full Hash: 0bffa7979736865a3e660f6d676ac829881d7b01
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-07-13 21:42:32
# Regressor Bug: 1718924
# File Overlap Count: 2
# Description:
#   Bug 1718924 - part 13: Move `TextServicesDocument::NodeHasOffsetEntry()` into `OffsetEntryArray` r=m_kato
#   
#   Depends on D119159
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119160
# ==============================================================================

diff -r 622d9feefc23 -r 0bffa7979736 editor/libeditor/EditorBase.cpp
--- a/editor/libeditor/EditorBase.cpp	Tue Jul 13 10:19:40 2021 +0000
+++ b/editor/libeditor/EditorBase.cpp	Tue Jul 13 10:46:40 2021 +0000
@@ -2220,7 +2220,7 @@
 
     if (mTextServicesDocument && NS_SUCCEEDED(rv)) {
       RefPtr<TextServicesDocument> textServicesDocument = mTextServicesDocument;
-      textServicesDocument->DidDeleteNode(&aContent);
+      textServicesDocument->DidDeleteContent(aContent);
     }
   } else {
     rv = NS_ERROR_FAILURE;
@@ -4771,10 +4771,10 @@
   if (mTextServicesDocument && NS_SUCCEEDED(rv) && deleteContent &&
       !deleteCharData) {
     RefPtr<TextServicesDocument> textServicesDocument = mTextServicesDocument;
-    textServicesDocument->DidDeleteNode(deleteContent);
+    textServicesDocument->DidDeleteContent(*deleteContent);
     MOZ_ASSERT(
         destroyedByTransaction || !Destroyed(),
-        "TextServicesDocument::DidDeleteNode() must not destroy the editor");
+        "TextServicesDocument::DidDeleteContent() must not destroy the editor");
   }
 
   if (!mActionListeners.IsEmpty() && deleteContent && !deleteCharData) {